charms/candid/pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
+log_cli = 1
+log_cli_level = INFO
+log_cli_format = %(asctime)s [%(levelname)8s] %(message)s (%(filename)s:%(lineno)s)
```

Comment:
This formatting has 8s spaces after the level name, probably kinda confusing to read? Would remove 8s in favour of just s, it looks fine in terminal

Reply:
done